bindings/generated/constantine_bls12_381.h

```diff
@@ -0,0 +1,80 @@
+/* Generated by Nim Compiler v1.4.6 */
+/*   (c) 2017 Andreas Rumpf */
+/* The generated code is subject to the original license. */
+#ifndef __constantine_bls12_381__
+#define __constantine_bls12_381__
+#define NIM_INTBITS 64
+
+/* section: NIM_merge_HEADERS */
+
+#include "nimbase.h"
+#undef LANGUAGE_C
+#undef MIPSEB
+#undef MIPSEL
+#undef PPC
+#undef R3000
+#undef R4000
+#undef i386
+#undef linux
+#undef mips
+#undef near
+#undef far
+#undef powerpc
+#undef unix
+
+/* section: NIM_merge_FRAME_DEFINES */
+
+/* section: NIM_merge_FORWARD_TYPES */
+typedef struct bls12381_fr bls12381_fr;
+typedef struct bls12381_fp bls12381_fp;
+
+/* section: NIM_merge_TYPES */
+
+/* section: NIM_merge_SEQ_TYPES */
+
+/* section: NIM_merge_FIELD_INFO */
+
+/* section: NIM_merge_TYPE_INFO */
+
+/* section: NIM_merge_PROC_HEADERS */
+N_LIB_PRIVATE N_NOCONV(void, signalHandler)(int sign);
+N_LIB_IMPORT N_CDECL(void, ctt_bls12381_fr_unmarshalBE)(bls12381_fr* dstX60gensym2_, NU8* srcX60gensym2_, NI srcX60gensym2_Len_0);
```

Comment:
The parameters name shouldn't be mangled, the N_LIB_IMPORT and N_CDECL hurts readability

Reply:
Also NU8* should be `char *` or `uint8_t*` and NI should be `ptrdiff_t`